playwright-tests/tests/asset-exchange/create-exchange-request.spec.js

```diff
@@ -0,0 +1,389 @@
+          },
+        },
+      },
+    });
+  });
+
+  test("create FT token swap request and display in pending request", async ({
+    page,
+    instanceAccount,
+    daoAccount,
+  }) => {
+    test.setTimeout(60_000);
+    await openCreatePage({ page, instanceAccount });
+    await selectSendToken({ page, token: "USDt" });
+    await page.frameLocator("iframe").locator("#send-amount").fill("1");
+    await selectReceiveToken({ page, token: "USDC" });
+    const calculateBtn = page.frameLocator("iframe").getByText("Calculate");
+    const submitBtn = page.frameLocator("iframe").getByText("Submit");
+    await expect(calculateBtn).toBeEnabled();
+    await expect(submitBtn).toBeDisabled();
+    const response = {
+      transactions: [
+        {
+          receiverId: "usdt.tether-token.near",
+          functionCalls: [
+            {
+              methodName: "ft_transfer_call",
+              args: {
+                receiver_id: "v2.ref-finance.near",
+                amount: "1000000",
+                msg: '{"force":0,"actions":[{"pool_id":5516,"token_in":"usdt.tether-token.near","token_out":"a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48.factory.bridge.near","amount_in":"1000000","amount_out":"0","min_amount_out":"0"},{"pool_id":4179,"token_in":"a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48.factory.bridge.near","token_out":"17208628f84f5d6ad33f0da3bbbeb27ffcb398eac501a31bd6ad2011e36133a1","amount_out":"0","min_amount_out":"989412"}]}',
+              },
+              gas: "180000000000000",
+              amount: "1",
+            },
+          ],
+        },
+      ],
+      outEstimate: "0.99940",
+    };
+    await mockSwapResponse({ page, response, daoAccount });
+    await calculateBtn.click();
+    await expect(submitBtn).toBeEnabled();
+    await submitBtn.click();
+    expect(await getTransactionModalObject(page)).toEqual({
+      proposal: {
+        description:
+          "* Proposal Action: asset-exchange <br>* Token In: usdt.tether-token.near <br>* Token Out: 17208628f84f5d6ad33f0da3bbbeb27ffcb398eac501a31bd6ad2011e36133a1 <br>* Amount In: 1 <br>* Slippage: 1 <br>* Amount Out: 0.99940",
+        kind: {
+          FunctionCall: {
+            actions: [
+              {
+                args: "eyJyZWNlaXZlcl9pZCI6InYyLnJlZi1maW5hbmNlLm5lYXIiLCJhbW91bnQiOiIxMDAwMDAwIiwibXNnIjoie1wiZm9yY2VcIjowLFwiYWN0aW9uc1wiOlt7XCJwb29sX2lkXCI6NTUxNixcInRva2VuX2luXCI6XCJ1c2R0LnRldGhlci10b2tlbi5uZWFyXCIsXCJ0b2tlbl9vdXRcIjpcImEwYjg2OTkxYzYyMThiMzZjMWQxOWQ0YTJlOWViMGNlMzYwNmViNDguZmFjdG9yeS5icmlkZ2UubmVhclwiLFwiYW1vdW50X2luXCI6XCIxMDAwMDAwXCIsXCJhbW91bnRfb3V0XCI6XCIwXCIsXCJtaW5fYW1vdW50X291dFwiOlwiMFwifSx7XCJwb29sX2lkXCI6NDE3OSxcInRva2VuX2luXCI6XCJhMGI4Njk5MWM2MjE4YjM2YzFkMTlkNGEyZTllYjBjZTM2MDZlYjQ4LmZhY3RvcnkuYnJpZGdlLm5lYXJcIixcInRva2VuX291dFwiOlwiMTcyMDg2MjhmODRmNWQ2YWQzM2YwZGEzYmJiZWIyN2ZmY2IzOThlYWM1MDFhMzFiZDZhZDIwMTFlMzYxMzNhMVwiLFwiYW1vdW50X291dFwiOlwiMFwiLFwibWluX2Ftb3VudF9vdXRcIjpcIjk4OTQxMlwifV19In0=",
```

Comment:
Would be easier to read this if you had a JSON object here that you stringify and base64 encode